src/Tools/BuildValidator/BuildValidator.csproj

```diff
@@ -2,17 +2,12 @@
 <!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
 
 <Project Sdk="Microsoft.NET.Sdk">
-  <Import Project="$(RepositoryEngineeringDir)targets\GenerateCompilerExecutableBindingRedirects.targets" />
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1;net472</TargetFrameworks>
     <PlatformTarget>AnyCPU</PlatformTarget>
-    <ServerGarbageCollection>true</ServerGarbageCollection>
-    <!-- https://github.com/dotnet/runtime/issues/1713 -->
-    <TieredCompilation>false</TieredCompilation>
-    <!-- Automatically generate the necessary assembly binding redirects -->
-    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
     <Nullable>enable</Nullable>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
Maybe I read right past it but where do we use `unsafe` code?

Reply:
https://github.com/RikkiGibson/roslyn/blob/rebuild-roslyn/src/Tools/BuildValidator/CompilationDiff.cs#L145